React InstantSearch Hooks/query-suggestions/src/App.css

```diff
@@ -0,0 +1,23 @@
+body,
+h1 {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica,
+    Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
+}
+
+.container {
+  max-width: 1200px;
+  margin: 0 auto;
+  padding: 1rem;
+}
+
+.current-refinements {
+  display: flex;
+  align-items: center;
+  padding: 1rem 0;
+  min-height: 64px;
```

Comment:
Is this really necessary? Disabling it seems to work just fine.

Reply:
It prevents a subtle layout shift when refinements are added / cleared.

![CleanShot 2022-06-30 at 15 32 27](https://user-images.githubusercontent.com/154633/176690353-7a312028-c98a-43a2-a451-0f5064b5532f.gif)